src/js/components/Button/button.stories.js

```diff
@@ -149,6 +150,32 @@ const ColoredButton = props => (
   </Grommet>
 );
 
+const customButtonColor = deepMerge(grommet, {
```

Comment:
I think `Grommet` is already doing a `deepMerge`?

Reply:
Grommet does deepMerge of the base theme. I want the font to be the grommet one. and enhance the rest. makes sense ?

Reply:
Ah. Thanks.